ephios/core/templates/core/fragments/event_list_list_mode.html

```diff
@@ -28,25 +28,37 @@
             {% with counter=event|event_list_signup_state_counts stats=event.get_signup_stats %}
                 <li class="list-group-item p-0 d-flex flex-row">
                     <div class="m-0 py-2 d-flex flex-column flex-lg-row-reverse justify-content-around flex-grow-0">
-                        <div class="ps-lg-2 d-flex flex-row flex-lg-column justify-content-center event-list-status-icon"
-                             {% if counter %}
-                                 data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-html="true"
-                                 title="{% for state in counter %}<div>
-                                            {% if event.shifts.all|length > 1 %}
-                                                {{ counter|get:state }} {% endif %}{{ ParticipationStates.labels_dict|get:state }}
-                                            </div>{% endfor %}"
-                             {% endif %}>
-                            {% if counter|get:ParticipationStates.CONFIRMED > 0 %}
-                                <span class="text-success fa fa-user-check ps-2"></span>
-                            {% elif counter|get:ParticipationStates.REQUESTED > 0 %}
-                                <span class="text-warning fa fa-user-clock ps-2"></span>
-                            {% elif counter|get:ParticipationStates.RESPONSIBLE_REJECTED > 0 %}
-                                <span class="text-danger fa fa-user-times ps-2"></span>
-                            {% elif counter|get:ParticipationStates.USER_DECLINED > 0 and counter|length == 1 %}
-                                <span class="text-danger fa fa-user-minus ps-2"></span>
-                            {% else %}
-                                <span class="text-body-secondary far fa-user ps-2"></span>
+                        <div class="ps-lg-2 d-flex flex-column justify-content-center">
+                            {% if event.can_change %}
```

Comment:
change color to info if can_change, move visible for groups to the right as a tooltip on the user-group icon.